tds-platform/build.gradle

```diff
@@ -80,22 +80,22 @@ dependencies {
     runtime 'org.apache.logging.log4j:log4j-web'
 
     // tds
-    api 'org.json:json:20180813'
+    api 'org.json:json:20230227'
     api 'com.coverity.security:coverity-escapers:1.1.1'
     api 'org.thymeleaf:thymeleaf-spring5:3.0.15.RELEASE'
     api 'javax.validation:validation-api:2.0.1.Final'
     api "org.hibernate.validator:hibernate-validator:${depVersion.hibernateValidator}"
     api "org.hibernate.validator:hibernate-validator-annotation-processor:${depVersion.hibernateValidator}"
     api 'javax.el:javax.el-api:3.0.0'
-    api 'org.glassfish:javax.el:3.0.0'
+    api 'org.glassfish:jakarta.el:4.0.2'
 
     // waterml stuff
     api 'org.apache.xmlbeans:xmlbeans'
     api 'org.n52.sensorweb:52n-xml-waterML-v20'
     api 'org.n52.sensorweb:52n-xml-om-v20'
 
     // edal-java (ncwms)
-    def edalVersion = '1.5.0.6-SNAPSHOT'
+    def edalVersion = '1.5.2'
```

Comment:
we don't was to use their version of edal-java, we need to keep using ours

Reply:
Yep! Fixing that now.